sites/feedback/ajax/page-feedback-contact-en.html

```diff
@@ -0,0 +1,71 @@
+		{ "selector": "html", "attr": "lang", "path": "language" },
+		{ "interface": "locationHref", "path": "submissionPage" },
+		{ "selector": "#wb-lng ul li:first-child a[lang]", "attr": "href", "path": "oppositelang" },
+		{ "selector": "[data-feedback-theme]", "attr": "data-feedback-theme", "path": "themeopt" },
+		{ "selector": "[data-feedback-section]", "attr": "data-feedback-section", "path": "sectionopt" },
+		{ "selector": "meta[name=\"dcterms.creator\"]", "attr": "content", "path": "institutionopt" },
+		{ "selector": "[data-feedback-link]", "attr": "data-feedback-link", "path": "contact/link" },
+		{ "selector": "[data-feedback-url]", "attr": "data-feedback-url", "path": "contact/url" }
+	]
+}'>
+	<div class="col-sm-10 col-md-9 col-lg-8">
+		<section class="well mrgn-bttm-0" aria-live="polite">
+			<h3 class="wb-inv">Give feedback about this page</h3>
+			<form action="https://pagesuccessemailqueue.azurewebsites.net/api/QueueProblemForm" method="post" class="wb-postback" data-wb-postback='{"success":".gc-pft-thnk"}'>
+				<div data-wb-json='{
+					"url": "#[feedback]",
+					"mapping": [
+						{ "selector": "input", "attr": "name", "value": "/@id" },
+						{ "selector": "input", "attr": "value", "value": "/@value" }
+					]
+				}'>
+					<template>
+						<input type="hidden" name="" value="" />
+					</template>
+				</div>
+				<fieldset class="gc-pft-btns chkbxrdio-grp row row-no-gutters d-sm-flex flex-sm-wrap align-items-sm-center">
+					<legend class="col-xs-12 col-sm-7 nojs-col-sm-12 col-md-9 col-lg-8 text-center text-sm-left nojs-text-left mrgn-tp-sm pr-sm-3 nojs-pr-sm-0"><span class="field-name">Did you find what you were looking for?</span></legend>
+					<div class="col-xs-12 nojs-show">
+						<button name="helpful" value="Yes-Oui" class="btn btn-primary" aria-describedby="why-note">Yes</button>
+					</div>
+					<div class="col-xs-12 col-sm-5 col-md-3 col-lg-4 text-center text-sm-right nojs-hide">
+						<button name="helpful" value="Yes-Oui" class="btn btn-primary mrgn-tp-sm mrgn-bttm-sm">Yes</button>
+						<button class="btn btn-primary mrgn-tp-sm mrgn-bttm-sm mrgn-lft-sm" data-wb-doaction='[
+							{"action":"removeClass","source":".gc-pft-no","class":"nojs-show"},
+							{"action":"addClass","source":".gc-pft-btns","class":"hide"}
+						]'>No</button>
+					</div>
+				</fieldset>
+				<div class="gc-pft-no nojs-show">
+					<p id="why-note" class="nojs-show mrgn-tp-lg mrgn-bttm-md"><strong>If not, tell us why below:</strong></p>
+					<p class="nojs-hide wb-inv"><strong>Tell us why below:</strong></p>
+					<details>
+						<summary><b>Need urgent help with a problem? Contact us</b></summary>
+						<p class="mrgn-bttm-0 mrgn-tp-md">
+							<a href="#" data-wb-json='[
+							{ "url": "#[feedback]/contact/url", "type": "attr", "attr": "href" },
+							{ "url": "#[feedback]/contact/link", "type": "replace" }
+							]'></a>
+						</p>
+					</details>
+					<div aria-live="off">
```

Comment:
Has been tested with AT?

```
aria-live="off"
```

Reply:
Removed `aria-live="polite"` to the component wrapper and removed `<div aria-live="off">` wrapping the content of the "No" answer. 

Adding `aria-live="polite"` to the first paragraph of the "No" answer and adding `role="status"` to the thank you message. This will announce the paragraph when user selects "No" and will announce the thank you message when user selects "Submit".